sktime/forecasting/croston.py

```diff
@@ -113,7 +115,7 @@ def _predict(
         forecast : pd.series
                    predicted forecasts
         """
-        if return_pred_int or X is not None:
+        if return_pred_int:
```

Comment:
Shouldn't it be the other way around? Keeping  
```suggestion
        if X is not None:
```

Reply:
actually, the entire "if X is not none" stuff must go.
If a forecaster ignores X, that's fine, but it shouldn't raise an error.